package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html

```diff
@@ -116,6 +130,17 @@ <h2><%:Overview%></h2>
 					<dt><%:Firmware%></dt><dd><%| nodeinfo.software.firmware.release %></dd>
 					<% if nodeinfo.software.fastd then -%>
 						<dt><%:Mesh VPN%></dt><dd><%| enabled(nodeinfo.software.fastd.enabled) %></dd>
+						<% if nodeinfo.network.mesh_vpn and nodeinfo.network.mesh_vpn.bandwidth_limit.enabled then -%>
+						<dt><%:Bandwidth limit%></dt>
+						<dd>
+							<% if nodeinfo.network.mesh_vpn.bandwidth_limit.egress then -%>
```

Comment:
@AiyionPrime Can't you do `(nodeinfo.network.mesh_vpn.bandwidth_limit.egress or -1) >= 0` here?

Reply:
Not sure what that does in lua. But I may read up on it.
In order to achieve what?

Reply:
`a=b or c` means `if b then a=b else a=c`
You use it here yourself: `factor = factor or 1`

My suggestion leads to not showing ingress or egress if it is not set...

Reply:
What was 0 meaning again? If it was "unlimited", too then there should be a `> 0 `...

Reply:
This node runs the current codebase.
http://lilienstra-e.n.ffh.zone/cgi-bin/status

Values are not shown, if the bandwidth limit is 0.
There should not need to be a need to change the code, I think it already does what you intend?

Reply:
Isn't -1 unlimited? That's the case I cover...

Reply:
in case egress or ingress is less than zero, the value is not added to the json struct.
And if the values are not there, they're sections are not shown.
There really is nothing to fix here, I think.
Please mark it as resolved when you agree.

https://github.com/freifunk-gluon/gluon/pull/2217/files#diff-c39d8b60296397950f7acc8f7b9015c5a7a37f4fb2a5ee7b09224477be5af678R69

Reply:
I can't resolve things.